packages/grid/x-data-grid-generator/package.json

```
@@ -45,7 +45,7 @@
   },
   "peerDependencies": {
     "@mui/icons-material": "^5.2.5",
-    "@mui/material": "^5.2.8",
+    "@mui/material": "^5.2.3",
```

It sounds like it should be 
```suggestion
    "@mui/material": "^5.4.1",
```

per https://github.com/mui/mui-x/issues/2998#issuecomment-1036337411